tests/lava/magma/runtime/test_context_manager.py

```diff
@@ -0,0 +1,114 @@
+import unittest
```

Comment:
This is missing a license header. Please add this at the top of the file and leave a blank line to the first import statement.
```python
# Copyright (C) 2021-22 Intel Corporation
# SPDX-License-Identifier: BSD-3-Clause
# See: https://spdx.org/licenses/
```